handlers/k8sconfig_handler.go

```diff
@@ -354,10 +352,16 @@ func registerK8sComponents(l logger.Handler, config []byte, ctx string) error {
 			return err
 		}
 		err = core.RegisterWorkload(content)
+		// Everytime a worload is registered successfully, the SeedApplication function will run and seed all the applications that can be handled with local provider.
+		if err == nil {
+			if l.config.Providers["None"] != nil {
```

Comment:
Instead receive `prov models.Provider` from the calling function and check `if prov.Name() == "None" { ...` as this can break somewhere, however interface won't.

Reply:
roger that